### ISSUE TITLE
need to be able to make it scriptable

### ISSUE THREAD

**Issue by despiegk** [threefoldtecharchive/zosctl#21]
e.g. 

```bash

export name="jumpscale"

#create container with name jumpscale, enable ssh
./zos container new --name=$name --root="https://hub.grid.tf/tf-official-apps/ubuntu-bionic-build.flist" --ssh

#if no id specified will be last one
zos container exec "curl https://raw.githubusercontent.com/threefoldtech/jumpscale_core/development_simple/install.sh?$RANDOM > /tmp/install_jumpscale.sh;bash /tmp/install_jumpscale.sh"
```

see #22

**Comment by xmonader** [comment]
solved in e377a3052a61a3f7d449d0aea28261873e92df04
![selection_036](https://user-images.githubusercontent.com/64129/46947522-5cb94280-d07b-11e8-9215-47fb69618da8.jpg)

**Comment by Dina-Abd-Elrahman** [comment]
# ZOS Version:
9a29aad058000d19d3278ac4c700b8761e00997c

# Verification Steps:
- ./zos container new --name=testcreate --root="https://hub.grid.tf/tf-official-apps/ubuntu-bionic-build.flist" --ssh
- ./zos container  exec 'apt-get update'
- ./zos container  exec 'apt-get install python3-pip'
